src/output/plugins/httpd/HttpdClient.cxx

```diff
@@ -41,6 +50,10 @@ HttpdClient::BeginResponse() noexcept
 {
 	assert(state != State::RESPONSE);
 
+	if (!head_method) {
```

Comment:
I don't think it's necessary to password protect HEAD requests, is it?

Reply:
You wrote extra code to implement a special case to allow HEAD requests. What justifies this extra complexity?
Your answer isn't good because extra complexity should be justified, not the other way round.